internal/service/cluster/resource_cluster_migration_test.go

```diff
@@ -85,40 +76,36 @@ func TestMigClusterRSCluster_basic_PartialAdvancedConf_backwardCompatibility(t *
 	})
 }
 
-func testAccMongoDBAtlasClusterConfigAdvancedConfPartialUpdated(orgID, projectName, name, autoscalingEnabled string, p *matlas.ProcessArgs) string {
+func configAdvancedConfPartialUpdated(projectID, name, autoscalingEnabled string, p *matlas.ProcessArgs) string {
 	return fmt.Sprintf(`
-		resource "mongodbatlas_project" "cluster_project" {
-			name   = %[2]q
-			org_id = %[1]q
-		}
-		resource "mongodbatlas_cluster" "advance_conf" {
-			project_id   = mongodbatlas_project.cluster_project.id
-			name         = %[3]q
+		resource "mongodbatlas_cluster" "test" {
+			project_id   = %[1]q
+			name         = %[2]q
 			disk_size_gb = 10
 
             cluster_type = "REPLICASET"
 		    replication_specs {
 			  num_shards = 1
 			  regions_config {
-			     region_name     = "EU_CENTRAL_1"
+			     region_name     = "US_WEST_2"
```

Comment:
I am curious about the change of region, why the change?

Reply:
to avoid cross-region limits